Rails/Ruby on Rails.sublime-syntax

```diff
@@ -87,10 +87,12 @@ contexts:
           pop: true
         - include: 'scope:source.ruby'
         - include: main
-    - match: \b(before_filter|skip_before_filter|skip_after_filter|after_filter|around_filter|filter|filter_parameter_logging|layout|require_dependency|render|render_action|render_text|render_file|render_template|render_nothing|render_component|render_without_layout|rescue_from|url_for|redirect_to|redirect_to_path|redirect_to_url|respond_to|helper|helper_method|model|service|observer|serialize|scaffold|verify|hide_action|append_view_path|prepend_view_path|view_paths)\b
+    - match: '\b(before_(filter|action)|skip_before_(filter|action)|skip_after_(filter|action)|after_(filter|action)|around_(filter|action)|filter|filter_parameter_logging|layout|require_dependency|render|render_action|render_text|render_file|render_template|render_nothing|render_component|render_without_layout|rescue_from|url_for|redirect_to|redirect_to_path|redirect_to_url|respond_to|helper|helper_method|model|observer|serialize|scaffold|verify|hide_action|append_view_path|prepend_view_path|view_paths)(?![?!:])\b'
```

Comment:
`(?![?!:])\b` will probably not function quite the way we want.

`\b` is a word boundary, so it won't trigger unless `?`, `!` or `:` are followed by a word character, or there is no symbol at the end of the method name.

Additionally, I presume the `(?![?!:])` is to handle Ruby 1.9 symbol keys, correct? If so, it should probably be: `\b(?![?!]?:)`. That will prevent matching the following: `after_filter!:`, `after_filter?:` and `after_filter:`. It will still match `after_filter :foo_bar`.